tests/snappi_tests/pfc/test_lossless_response_to_external_pause_storms.py

```diff
@@ -4,11 +4,11 @@
     fanout_graph_facts_multidut     # noqa: F401
 from tests.common.snappi_tests.snappi_fixtures import snappi_api_serv_ip, snappi_api_serv_port, \
     get_snappi_ports_single_dut, snappi_testbed_config, \
-    get_snappi_ports_multi_dut, is_snappi_multidut, \
+    get_snappi_ports_multi_dut, is_snappi_multidut, snappi_port_selection, tgen_port_info, \
     snappi_api, snappi_dut_base_config, get_snappi_ports, get_snappi_ports_for_rdma, cleanup_config  # noqa: F401
 from tests.common.snappi_tests.qos_fixtures import prio_dscp_map, \
     lossless_prio_list, disable_pfcwd                                                       # noqa: F401
-from tests.snappi_tests.files.helper import multidut_port_info, setup_ports_and_dut, reboot_duts  # noqa: F401
+from tests.snappi_tests.files.helper import setup_ports_and_dut, reboot_duts  # noqa: F401
```

Comment:
```suggestion
from tests.snappi_tests.files.helper import  reboot_duts  # noqa: F401
```
I also have to make this changes here as well to not trigger the setup_ports_and_dut default fixture